### ISSUE TITLE
test 1.9.3: fail Bitmetv due to match selector

### ISSUE THREAD

**Issue by aroundmyroom** [cardigann/cardigann#184]
→ Indexer bitmetv FAILED with Failed to match selector "span.smallfont > font[color=\"#000000\"]"
ERRO[0037] Test failed                                   error=Failed to match selector "span.smallfont > font[color=\"#000000\"]"

DEBU[0031] Checking page test block                      path=https://www.bitmetv.org/messages.php selector= site=bitmetv
DEBU[0031] Resolving url                                 base=http://www.bitmetv.org/login.php?returnto=%2Fmessages.php site=bitmetv u=https://www.bitmetv.org/messages.php
DEBU[0031] Opening page                                  site=bitmetv url=https://www.bitmetv.org/messages.php
DEBU[0032] Finished request                              code=200 page=https://www.bitmetv.org/messages.php site=bitmetv
DEBU[0032] Successfully logged in                        site=bitmetv
DEBU[0032] Resolving url                                 base=https://www.bitmetv.org/messages.php site=bitmetv u=https://www.bitmetv.org/browse.php
DEBU[0032] Opening page                                  site=bitmetv url=https://www.bitmetv.org/browse.php
DEBU[0037] Finished request                              code=200 page=https://www.bitmetv.org/browse.php site=bitmetv
DEBU[0037] Ratio returned
FAILURE ✗ in 7.780718391s
→ Indexer bitmetv FAILED with Failed to match selector "span.smallfont > font[color=\"#000000\"]"
ERRO[0037] Test failed                                   error=Failed to match selector "span.smallfont > font[color=\"#000000\"]"

**Comment by kaso17** [comment]
please post the HTML code of your statistics (top left line).

it should look like this:
```
<span class="smallfont">Welcome back, <b><a href="/userdetails.php?id=123456">kaso17</a></b>&nbsp; [<a href="/logout.php">logout</a>]&nbsp;&nbsp; <b><font color="#8B4513">Connectable:&nbsp;</font><b><font color="green"><a title="Connectable = Yes">Yes</a></font></b>&nbsp;&nbsp;</b><br>

      <font color="1900D1">Ratio:</font> <font color="#000000">322.432</font>&nbsp;&nbsp;<font color="green">UL:</font> <font color="black">831.68 GB</font>&nbsp;&nbsp;<font color="darkred">DL:</font> <font color="black">3.15 GB</font>&nbsp;&nbsp;<font color="1900D1">Act:&nbsp;</font></span>
```

**Comment by kaso17** [comment]
It was caused by your <1 ratio, fixed with #187
